main.go

```diff
@@ -155,6 +197,79 @@ func findFile(r *zip.ReadCloser, name string) *zip.File {
 	return nil
 }
 
+func listAttributes(path string) {
+	if strings.HasSuffix(path, ".apk") || strings.HasSuffix(path, ".aab") {
+		fmt.Println("Listing attributes for", path)
+		listAttributesInZip(path)
+	} else {
+		fmt.Println("Listing attributes for AndroidManifest.xml")
```

Comment:
The actual `path` might not be AndroidManifest.xml. Maybe it's not even worth printing this info at all? You'd just repeat the same input the user has provided. That's like `ls` saying `Listing files in ...`. Instead, it just lists the files without announcing what it will do.